SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -115,13 +145,21 @@ void USpatialWorkerConnection::SendComponentUpdate(Worker_EntityId EntityId, FWo
 Worker_RequestId USpatialWorkerConnection::SendCommandRequest(Worker_EntityId EntityId, Worker_CommandRequest* Request, uint32_t CommandId,
 															  const TOptional<Trace_SpanId>& SpanId)
 {
-	check(Coordinator.IsValid());
```

Comment:
Why remove the check?

Reply:
good question. that must have happened during the last big rebase on top of master and I missed it